yb-voyager/src/callhome/diagnostics.go

```diff
@@ -94,7 +94,7 @@ type TargetDBDetails struct {
 type UnsupportedFeature struct {
 	FeatureName      string `json:"FeatureName"`
 	ObjectCount      int    `json:"ObjectCount"`
-	TotalOccurrences int    `json:"TotalOccurrences"`
```

Comment:
If we're filling it in everywhere, don't see a point for omitempty. Shall we remove it?